.github/workflows/main.yml

```diff
@@ -31,18 +34,19 @@ jobs:
       run: |
         poetry install
 
-    - name: Test with pytest
-      run: |
-        poetry run pytest --cov
-
     - name: Lint with mypy
       run: |
         poetry run mypy mousebender
-
+  
```

Comment:
```suggestion

```

You really need to set up your editor to strip out extraneous whitespace. 😆 Otherwise we are going to set up something like pre-commit and have it's trailing whitespace and/or YAML checker to tell you to stop it. 😉

Reply:
Done. And editor settings updated :muscle: